src/services/study/index.ts

```diff
@@ -167,32 +196,32 @@ export function getAvailableComponentLibraries() {
     console.info('get available component libraries for diagrams');
     const getAvailableComponentLibrariesUrl = PREFIX_STUDY_QUERIES + '/v1/svg-component-libraries';
     console.debug(getAvailableComponentLibrariesUrl);
-    return backendFetchJson(getAvailableComponentLibrariesUrl);
+    return backendFetchJson(getAvailableComponentLibrariesUrl) as Promise<string[]>;
```

Comment:
maybe define the function return type instead of this 'as ...' code.
`export function getAvailableComponentLibraries() : Promise<string[]>`